cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -41,57 +36,83 @@ context("Researcher uses permission forms", () => {
-        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
-        pf.clickCreateEditPermissionFormCancelButton();
-
-        pf.clickPermissionFormEdit(TEST_PERMISSION_FORM_NAME+'1');
-        pf.verifyEditPermissionFormDialog(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
-        pf.clickCreateEditPermissionFormCancelButton();
-
-        // TODO: add more testing of the Edit Permission Form Dialog (without clicking the 'Cancel' button)
-
-        cy.log('Test the Delete link, for an Archived, an Unarchived and a never-Archived Permision Form');
-        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'2');
-        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'3');
-        pf.clickPermissionFormDelete(TEST_PERMISSION_FORM_NAME+'4');
+        cy.log('Confirm the expected Permission Form names are present');
+        pf.verifyPermissionFormName(TEST_PERMISSION_FORM_NAME+'1');
+        pf.verifyPermissionFormName(TEST_PERMISSION_FORM_NAME+'2');
+        pf.verifyPermissionFormName(TEST_PERMISSION_FORM_NAME+'3');
+        pf.verifyPermissionFormName(TEST_PERMISSION_FORM_NAME+'4');
+        pf.verifyPermissionFormName(TEST_PERMISSION_FORM_NAME+'5');
+
+        cy.log('Test the Edit link & dialog, for a never-Archived, an Archived, and an Unarchived Permision Form; but click Cancel');
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            OTHER_TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1_cancel_changes', 'cancel_changes.'+TEST_PERMISSION_FORM_URL, true);
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            OTHER_TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2_cancel_changes', 'cancel_changes.'+TEST_PERMISSION_FORM_URL, true);
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
+            OTHER_TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3_cancel_changes', 'cancel_changes.'+TEST_PERMISSION_FORM_URL, true);
+        
+        cy.log('Test the Edit link & dialog, for a never-Archived, an Archived, and an Unarchived Permision Form; this time, Save Changes');
+        pf.selectCreateManagePermissionsProject();
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            null, TEST_PERMISSION_FORM_NAME+'1_modified', 'modified.'+TEST_PERMISSION_FORM_URL);
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            OTHER_TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2_modified', 'modified.'+TEST_PERMISSION_FORM_URL);
+        pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
+        pf.editPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
```

Comment:
We covered the scenario to ensure the changes in the edit dialog. Shall we also add the scenario to verify the changes are reflected fine in the **Create/ Manage Project Permission Forms** section after clicking on the Save Changes button in the dialog?